ods_ci/utils/scripts/util.py

```diff
@@ -69,17 +68,16 @@ def execute_command(cmd):
             encoding="utf-8",
             errors="replace",
         ) as p:
-            while True:
-                line = p.stdout.readline()
-                if line != "":
-                    output += line + "\n"
-                elif p.poll() is not None:
-                    break
-            sys.stdout.flush()
-            log.info(f"OUTPUT: {output}")
-            return output
-    except:
-        return None
+            output = []
+            for line in p.stdout:
+                output.append(line)
+                if print_stdout:
+                    print(">:", line, end="")
+                    sys.stdout.flush()
+            return "".join(output)
+    except Exception as e:
+        log.exception("Starting the subprocess failed", exc_info=e)
```

Comment:
Tentative (optional):

```suggestion
        log.exception(f"Starting the subprocess '{cmd}' failed", exc_info=e)
```

Reasoning: I know that we print the command right before this try block. I just think it's nice to have the command also as part of the exception message to be completely clear on what command was about to be executed as a subprocess. E.g. in case that there is some other output in the log in the meantime for any reason.

Reply:
If I recall correctly, the command that was executed is also logged in the exception data, but I'm not certain this is the case. No harm printing it twice, in the worst case.